backend/src/WebApi/Controllers/ClientController.cs

```diff
@@ -8,27 +11,50 @@ namespace PartyKlinest.WebApi.Controllers
     public class ClientController : ControllerBase
     {
         private readonly ILogger<ClientController> _logger;
+        private readonly ClientFacade _clientFacade;
 
-        public ClientController(ILogger<ClientController> logger)
+        public ClientController(ILogger<ClientController> logger, ClientFacade clientFacade)
         {
             _logger = logger;
+            _clientFacade = clientFacade;
         }
 
         /// <summary>
         /// Deletes account from the system.
         /// </summary>
-        /// <param name="id">Client's id.</param>
+        /// <param name="clientId">Client's id.</param>
         /// <returns></returns>
         /// <response code="200">OK</response>
         /// <response code="403">Not Authorized</response>
         [HttpDelete("{id}")]
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
-        public IActionResult DeleteClient(int id)
+        [Authorize(Policy = "AdminOnly")]
+        public async Task<IActionResult> DeleteClient(string clientId)
         {
-            _logger.LogInformation($"Delete client {id}");
-            return Ok();
+            _logger.LogInformation($"Delete client {clientId}");
+            if (User.IsCleaner())
```

Comment:
User to osoba wywołująca endpoint nie osoba reprezentująca clientId. Poza tym czemu w endpoincie /client/id jest cleaner?

Reply:
Nwm. Już poprawione